wagtailflags/templates/wagtailflags/includes/flag_index.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
+{% load wagtailflags_admin %}
+{% load i18n feature_flags flags_debug wagtailflags_admin %}
```

Comment:
Does this load wagtailflags_admin twice?

Reply:
Indeed so.